code/modules/antagonists/changeling/powers/tiny_prick.dm

```diff
@@ -122,8 +122,8 @@
 	dna_cost = 1
 
 /obj/item/melee/arm_blade/false
-	desc = "A grotesque mass of flesh that used to be your arm. Although it looks dangerous at first, you can tell it's actually quite dull and useless."
-	force = 5 //Basically as strong as a punch
+	desc = "A grotesque mass of flesh that used to be your arm. On the bright side, at least you can cut wood with this."
+	force = 30 //Prevents dual-stinging people with this sting to render them defenseless

```

Comment:
You wanna modularize this some? Maybe at least plop down some ``//yogs - fakeblade balance change``?

https://github.com/yogstation13/Yogstation-TG/blob/master/yogstation/README.md

Reply:
there i did it